lib/config.js

```diff
@@ -0,0 +1,94 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+var convict = require('convict');
+var fs = require('fs');
+var path = require('path');
+
+var conf = module.exports = convict({
+  env: {
+    doc: 'The current node.js environment',
+    default: 'prod',
+    format: [ 'dev', 'test', 'stage', 'prod' ],
+    env: 'NODE_ENV'
+  },
+  log: {
+    level: {
+      default: 'info',
+      env: 'LOG_LEVEL'
+    },
+    format: {
+      default: 'pretty',
+      format: ['heka', 'pretty'],
+      env: 'LOG_FORMAT'
+    },
+    app: {
+      default: 'fxa-basket-proxy',
+      env: 'LOG_APP_NAME'
+    }
+  },
+  publicUrl: {
+    doc: 'Url at which the server will be exposed',
+    format: 'url',
+    default: 'http://127.0.0.1:1114',
+    env: 'PUBLIC_URL'
+  },
+  oauthVerifyUrl: {
+    doc: 'Url at which to verify OAuth tokens',
+    format: 'url',
+    default: 'http://localhost:9010/v1/verify',
+    env: 'OAUTH_VERIFY_URL'
+  },
+  corsOrigin: {
+    doc: 'Origins to allow in CORS headers',
+    default: '*',
```

Comment:
It's an authentictaed API so I believe `*` to be OK;  the auth ad profile servers also specify `*` FWIW.  Honestly, I put it to this because it makes things easier running locally, we could certainly lock it down to just content-server in production config.